javascript/net/grpc/web/generator/grpc_generator.cc

```diff
@@ -944,12 +959,16 @@ void PrintProtoDtsFile(Printer* printer, const FileDescriptor* file) {
 void PrintFileHeader(Printer* printer, const std::map<string, string>& vars) {
   printer->Print(
       vars,
+      "// Code generated by protoc-gen-grpc-web. DO NOT EDIT.\n"
```

Comment:
Do you mind provide an example output in the PR description? Thanks! :)

Reply:
Added example output. I only now noticed that this line wasn't the first line in the file. In the Go community it's a convention to place this line first, so that tools only need to check the first line.

Reply:
Ahh i see.. yeahh unfortunately the code will be passed through closure compiler so this cannot be at the top :)